src/context/WalletProvider/NativeWallet/components/NativeCreate.tsx

```diff
@@ -15,14 +16,15 @@ import { GENERATE_MNEMONIC, Vault } from '@shapeshiftoss/hdwallet-native-vault'
 import { range } from 'lodash'
 import { Component, useEffect, useMemo, useRef, useState } from 'react'
 import { FaEye } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
 import { Text } from 'components/Text'
 
 import { NativeSetupProps } from '../types'
 
-const getVault = async (): Promise<Vault> => {
+const getVault = async (mnemonic?: string): Promise<Vault> => {
```

Comment:
Call this `vault`. We won't be passing a mnemonic around, we'll create a wallet vault, then pass the vault through and then save the vault once we get a name and encryption password for the vault.